guides/source/active_record_querying.md

```diff
@@ -454,14 +454,22 @@ Client.where("orders_count = ?", params[:orders])
 
 Active Record will take the first argument as the conditions string and any additional arguments will replace the question marks `(?)` in it.
 
-If you want to specify multiple conditions:
+If you want to specify multiple conditions, you can pass multiple arguments:
 
 ```ruby
 Client.where("orders_count = ? AND locked = ?", params[:orders], false)
 ```
 
 In this example, the first question mark will be replaced with the value in `params[:orders]` and the second will be replaced with the SQL representation of `false`, which depends on the adapter.
 
+When you want to use a wildcard character (for example `%`), you can add the wildcard to the argument:
+
+```ruby
+Client.where("first_name LIKE ?", "%#{params[:first_name]}%")
```

Comment:
This query still allows the user to inject their own wildcard characters. [`sanitize_sql_like`](https://api.rubyonrails.org/v6.0.2.1/classes/ActiveRecord/Sanitization/ClassMethods.html#method-i-sanitize_sql_like) exists for this use case:

```suggestion
Client.where("first_name LIKE ?", "%#{Client.sanitize_sql_like(params[:first_name])}%")
```

In my opinion this example is too complicated, though. It's not immediately clear what `sanitize_sql_like` is doing versus the `?` placeholder, and the explanation for _why_ values need to be escaped at all isn't until the next section.

Perhaps we should instead change the `LIKE` query in the warning earlier in the guide to a simpler example.

Reply:
Ah, I didn't think of the ability to inject wildcard characters. I always thought my example was the preferred example. This is also why I'd like to see an example in the guides, instead of having all kinds of solutions on Stackoverflow. Or maybe we can reintroduce `where.like`? 😉 

I'll simplify the warning example and move the like example with `sanitize_sql_like ` to the Argument Safety section.